pennylane/transforms/metric_tensor.py

```diff
@@ -37,6 +37,16 @@ def expand_fn(tape, approx=None, allow_nonunitary=True, aux_wire=None, device_wi
 def metric_tensor(tape, approx=None, allow_nonunitary=True, aux_wire=None, device_wires=None):
     r"""Returns a function that computes the metric tensor of a given QNode or quantum tape.
 
+    The metric tensor convention we employ here has the following form:
+
+    .. math::

```

Comment:
@dwierichs can you make sure I am not missing a factor 2, please!

Reply:
I'll take a look!

Reply:
Yes, this is indeed correct :)